third_party/open-policy-agent/gatekeeper/helmify/static/values.yaml

```
@@ -0,0 +1,48 @@
+image:
+  repository: ghcr.io/azure/eraser-manager
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+workerImage:
```

```suggestion
eraser: 
	image:
```

let's move this one level below